vhost/src/vhost_user/connection.rs

```diff
@@ -275,6 +275,42 @@ impl<H: MsgHeader> Endpoint<H> {
         Ok(())
     }
 
+    /// Send a message with header, body and payload. Optional file descriptors
+    /// may also be attached to the message. Unlike send_message_with_payload
+    /// allows sending messages bigger than MAX_MSG_SIZE (up to MAX_OVERSIZED_MSG_SIZE)
+    /// # Return:
+    /// * - number of bytes sent on success
+    /// * - SocketRetry: temporary error caused by signals or short of resources.
+    /// * - SocketBroken: the underline socket is broken.
+    /// * - SocketError: other socket related errors.
+    /// * - OversizedMsg: message size is too big.
+    /// * - PartialMessage: received a partial message.
+    /// * - IncorrectFds: wrong number of attached fds.
+    pub fn send_message_oversized_with_payload<T: ByteValued>(
```

Comment:
Please split this commit in 2, one where you add send_message_oversized_with_payload() related stuff and one for update_scanout().

Also, if the oversized is supported only by gpu messages, can we move these new API there?